libsql/src/local/impls.rs

```diff
@@ -15,7 +15,8 @@ pub(crate) struct LibsqlConnection {
     pub(crate) conn: super::Connection,
 }
 
-#[async_trait::async_trait]
+#[cfg_attr(feature = "hrana-cf", async_trait::async_trait(?Send))]
+#[cfg_attr(not(feature = "hrana-cf"), async_trait::async_trait)]
```

Comment:
I don't think this is the right approach we should take since we need these features to be additive.